lib/src/widgets/yaru_radio_theme.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+@immutable
+class YaruRadioThemeData extends ThemeExtension<YaruRadioThemeData>
+    with Diagnosticable {
+  const YaruRadioThemeData({
+    this.color,
+    this.borderColor,
+    this.checkmarkColor,
+    this.indicatorColor,
+  });
+
+  final MaterialStateProperty<Color?>? color;
+  final MaterialStateProperty<Color?>? borderColor;
+  final MaterialStateProperty<Color?>? checkmarkColor;
+  final MaterialStateProperty<Color?>? indicatorColor;
```

Comment:
does a radio button have a "checkmark"? :thinking: is "indicator" the background?

Reply:
On the radio checkmark I'm pretty sure of myself, because it is something (like a dot or an 'ok' icon) who mark the togglable as checked :)

About the indicator, it is the half-transparent circle who shown when the togglable is hovered and/or focused.
Maybe rename it in stateIndicator? Because hoverFocusIndicator is maybe a bit too long imo.

Reply:
np, it's fine to keep it. :)